package.json

```diff
@@ -50,9 +60,11 @@
     "build": "tsup --minify --clean",
     "watch": "tsup --watch --sourcemap",
     "test": "npm run lint && npm run test-e2e",
-    "test-e2e": "npm run test-e2e:basic && npm run test-e2e:settings-view",
+    "test-e2e": "npm run test-e2e:basic && npm run test-e2e:settings-view && npm run test-e2e:vscode-settings && npm run test-e2e:empty",
     "test-e2e:basic": "wdio run ./tests/e2e/basic/wdio.conf.ts",
     "test-e2e:settings-view": "wdio run ./tests/e2e/settings-view/wdio.conf.ts",
+    "test-e2e:vscode-settings": "wdio run ./tests/e2e/vscode-settings/wdio.conf.ts",
```

Comment:
Here I test ability to use `configPath` specified in `.vscode/settings.json`

Reply:
Maybe we should use "concurrently" npm package here in order to run these in parallel?
Example: https://github.com/gemini-testing/testplane/blob/master/package.json#L15